packages/cli/project.json

```
@@ -40,6 +36,24 @@
         "passWithNoTests": true,
         "reportsDirectory": "../../coverage/packages/cli"
       }
+    },
+    "run-help": {
+      "command": "npx dist/packages/cli --help",
```

Should be the command not the argument
```suggestion
      "command": "npx dist/packages/cli help",
```